react/FilterNavigator.js

```diff
@@ -90,6 +95,7 @@ const FilterNavigator = ({
   fullWidthOnMobile = false,
   navigationTypeOnMobile = 'page',
   updateOnFilterSelectionOnMobile = false,
+  filtersDrawerDirectionMobile = DRAWER_DIRECTION_MOBILE.left,
```

Comment:
What do you think about this prop name: `drawerDirectionMobile`?

Reply:
For me it's the same. I changed it.